tests/tests_app/core/test_lightning_api.py

```diff
@@ -446,6 +447,27 @@ def test_configure_api():
         f"http://localhost:{APP_SERVER_PORT}/api/v1/request", data=InputRequestModel(name="hello").json()
     )
     assert response.json() == {"name": "hello", "counter": 1}
+
+    with open(__file__, "rb") as f:
```

Comment:
I find it strange to use the file we are testing in. Maybe use a tempfile instead?

Reply:
I tried but didn't manage to do so. I thought it didn't manage more.